CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+v0.8.0
+------
+* **Release date**: 2016-03-04.
+* Fixed migration loop to handle the case where there are no explicit migrations to run.
+* Railway route extraction no longer relies on the `planet_osm_nodes` table, which may be missing if flat nodes is enabled.
+* Add labels for bay water features
+* Add inclusive pois
+* Include all z17-18 pois at z16
+* Return min_zoom from pois query
+* Move building addresses to z16
```

Comment:
We also moved Fences to z16 (was z17).

For a different release, we should consider marking the addresses with `min_zoom:17`.